src/logger/loggerTaskEx.c

```diff
@@ -243,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
Changing this particular logic may cause a regression. Suggest moving to 2.8.6 where we can look at it more carefully.